tracecat/integrations/router.py

```diff
@@ -484,7 +484,7 @@ async def delete_integration(
 
     # For custom providers, delete the provider definition even if no integration exists
     if integration is None:
-        if provider_info.key.id.startswith("custom-"):
+        if provider_info.key.id.startswith(("custom_", "custom-")):
```

Comment:
This might be unexpected behavior for the user. Can we make the user manually delete the old integration instead?